storage_media/models/storage_media.py

```diff
@@ -29,14 +24,15 @@ def onchange_name(self):
         for record in self:
             if record.name:
                 filename, extension = os.path.splitext(record.name)
-                record.name = f"{slugify(filename)}{extension}"
-
-    @api.model
-    def create(self, vals):
-        vals["file_type"] = self._default_file_type
-        if "backend_id" not in vals:
-            vals["backend_id"] = self._get_default_backend_id()
-        return super(StorageMedia, self).create(vals)
+                record.name = f"{self.env['ir.http']._slugify(filename)}{extension}"
```

Comment:
ok, could you point to `https://github.com/odoo/odoo/pull/175041`, mention it in the PR's description?

Reply:
hehe, ok. FYI, We have a sheet to note the changes and PR changes. You don't need to start over.

Reply:
ok, but part of the reasons to do so is to show them we understand the change and avoid being challenged by external reviewers, right? 

How can they access our internal sheet? :)